.travis.yml

```diff
@@ -1,14 +1,12 @@
 dist: xenial
 language: ruby
-bundler_args: "--without development"
```

Comment:
since you moved `webmock` to tests, you can reinstate this and save a couple of seconds on travis bootup time